src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogHttpClient.cs

```diff
@@ -55,7 +55,7 @@ private List<JsonPayloadBuilder> BuildEvents(IEnumerable<LogEvent> events)
                 var payloads = _renderer.RenderDatadogEvents(logEvent);
                 foreach (var payload in payloads)
                 {
-                    if (builder.Size() >= _maxPayloadSize || builder.Count() >= _maxMessageCount)
+                    if (builder.Size()+payload.Length >= _maxPayloadSize || builder.Count() >= _maxMessageCount)
```

Comment:
I think this should be `Encoding.UTF8.GetByteCount(payload)` to get the byte length rather than `Length` which is number of characters?

Reply:
That is actually a great remark, good catch thank you.